src/main/resources/samplePatient.xsd

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" targetNamespace="http://sample.dto.icellmobilsoft.hu/patient" xmlns="http://sample.dto.icellmobilsoft.hu/patient" elementFormDefault='qualified'>
+    <xsd:complexType name="departmentType">
+        <xsd:sequence>
+            <xsd:element name="id" type="xsd:string" />
+            <xsd:element name="name" type="xsd:string" />
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:complexType name="instituteType">
+        <xsd:sequence>
+            <xsd:element name="id" type="xsd:string" />
+            <xsd:element name="name" type="xsd:string" />
+            <xsd:element name="departments">
+                <xsd:complexType>

```

Comment:
elementek esetén nem használunk, így beágyazva complexType-ot , ezt emeled ki külön és 'type'-nál használd a létrehozott complexType-ot, legyen a neve DepartmentIdType, aminek a típusa EntityIdType